Improve Ergo Jr page.

pierre-rouanet opened the pull request:
I've added some information on the Ergo Jr page to work on #22 . Let me know what you think.

@damiencaselli I did not succeed to integrate the assembly gif.

pierre-rouanet committed:
Add information on the Ergo Jr page.

pierre-rouanet committed:
Add snap example image.

pierre-rouanet commented:
I also don't really get why travis is unhappy: https://travis-ci.org/poppy-project/poppy-project.org/builds/133949837#L240

> -  error parsing attribute name (line 1)

damiencaselli commented on _pages/robots/ergo/en.html:
```diff
@@ -24,3 +24,42 @@
+<div class="video-showcase">
+  <div class="row">
+    <div class="large-6 columns">
+      <div class="flex-video"->
```
`<div class="flex-video"->` should be ` <div class="flex-video">`

pierre-rouanet committed:
Typos.

pierre-rouanet committed:
Use Youcef videos instead.

dashdashzako committed:
Update Ergo Jr draft

ghost commented:
@pierre-rouanet I've reviewed text, good job 👍 
I'll add this to the master and translate it to French.

dashdashzako committed:
Update Ergo Jr page

French translation is done.
The robot is now mentioned as Ergo Jr instead of Poppy Ergo Jr, in
order to have consistent naming conventions and to avoid confusion
about Poppy.

pierre-rouanet commented:
Thanks for the review @damiencaselli. I'm agree with all of your improvements!
